scala/scala.bzl

```diff
@@ -34,25 +34,38 @@ def _get_all_runfiles(targets):
       runfiles += _get_runfiles(target)
     return runfiles
 
-def _adjust_resources_path(path):
+
+def _adjust_resources_path_by_strip_prefix(path,resource_strip_prefix):
+    if not path.startswith(resource_strip_prefix):
+      fail("Resource file %s is not under the specified prefix to strip" % path)
```

Comment:
you're right. I didn't test it (will do). The motivation is to be compliant with what happens with java rules and scala rules (that have sources)